sylt-compiler/src/compiler.rs

```diff
@@ -1265,17 +1264,25 @@ impl Compiler {
             for statement in module.statements.iter() {
                 use StatementKind::*;
                 match &statement.kind {
-                    Use { file: Identifier { name, span }, file_alias, .. } => {
-                        let use_path = root.join(format!("{}.sy", name));
-                        let other = path_to_namespace_id[&use_path];
+                    Use { file: Identifier { name, span }, file_alias, resolved_path } => {
+                        let bare_name = name.trim_end_matches("/");
+                        let other = path_to_namespace_id[resolved_path];
```

Comment:
Move this down - it can be placed after the match block and remove some clutter.